esm-samples/README.md

```diff
@@ -8,15 +8,19 @@ Run `npm install` in a project directory and then start adding ArcGIS API for Ja
 
 For additional information, see the [Build with ES modules](https://developers.arcgis.com/javascript/latest/es-modules/) Guide topic in the SDK.
 
+### TypeScript
+
+For TypeScript users, the type definitions are included when you install `@arcgis/core`. There is no need for a separate install.
 
 ### Bundle size and performance
 
 For each release, we track certain metrics including bundle size and web application performance. These files can be found in the [/esm-samples/.metrics](./.metrics/) directory. You can read more about how we capture this information this [README](../.github/scripts/README.md).
 
-Bundle size is dependant on multiple factors and will vary based on framework, module bundler, transpiling and related configurations, as well as the ArcGIS API for JavaScript functionality that has been implemented. The compile process can output several hundred or more bundles on-disk due to using dynamic imports. At runtime, your application will typically only load a portion of the bundles depending on the `@arcgis/core` functionality used. 
+Bundle size is dependant on multiple factors and will vary based on framework, module bundler, transpiling and related configurations, as well as the ArcGIS API for JavaScript functionality that has been implemented. The compile process can output several hundred or more bundles on-disk due to using dynamic imports. At runtime, your application will typically only load a portion of the bundles depending on what `@arcgis/core` modules are used in your project. 
```

Comment:
How about one of these?
- depending on the `@arcgis/core` modules used in your project.
- depending on which `@arcgis/core` modules are used in your project.

Reply:
Done, I went with "which".